ERROR in ./~/angular2-highcharts/dist/Highcharts.js

After pulling the latest version 0.3.3 , I was getting this error stack 

```
ERROR in ./~/angular2-highcharts/dist/Highcharts.js
Module not found: Error: Can't resolve 'highcharts/highstock.src' in '/Users/gaurav.pandvia/minjar/git/newCode/bm_dashbaord/node_modules/angular2-highcharts/dist'
 @ ./~/angular2-highcharts/dist/Highcharts.js 2:17-52
 @ ./~/angular2-highcharts/dist/index.js
 @ ./~/angular2-highcharts/index.js
 @ ./src/app/xyz.module.ts
 @ ./src/app/xyz.routing.ts
 @ ./src/app/zzzz.module.ts
 @ ./src/app/app.module.ts
 @ ./src/main.browser.ts
 @ multi main
```

The fix I made was to include 'highcharts : 4.2.7' inside my package.json. @gevgeny @Fank  Did you deliberately removed it or if it is missed out in your package.json then please include it

--- a/package.json
+++ b/package.json
@@ -50,6 +50,7 @@
     "zone.js": "0.6.25"
   },
   "dependencies": {
-    "@types/highcharts": "^4.2.36"
+    "@types/highcharts": "^4.2.36",
+    "highcharts": "^5.0.0"
   }
 }

--- a/package.json
+++ b/package.json
@@ -51,6 +51,6 @@
   },
   "dependencies": {
     "@types/highcharts": "^4.2.36",
-    "highcharts": "^5.0.0"
+    "highcharts": "^4.2.7"
   }
 }

